retrofit2/src/main/java/com/linecorp/armeria/client/http/retrofit2/ArmeriaRetrofitBuilder.java

```diff
@@ -71,15 +70,13 @@
  */
 public final class ArmeriaRetrofitBuilder {
 
-    private static final Pattern GROUP_PREFIX_PATTERN = Pattern.compile("^[_0-9a-z]+$");
     private static final BiFunction<String, ? super ClientOptionsBuilder, ClientOptionsBuilder>
             DEFAULT_CONFIGURATOR = (url, optionsBuilder) -> optionsBuilder;
     private static final String SLASH = "/";
 
     private final Retrofit.Builder retrofitBuilder;
     private final ClientFactory clientFactory;
     private String baseUrl;
-    private String basePath;
```

Comment:
No worries. I'd like to recommend [importing our IntelliJ IDEA inspection profile](https://github.com/line/armeria/blob/master/CONTRIBUTING.md#intellij-idea) and go through all the inspection warnings by [pressing F2](https://www.jetbrains.com/help/idea/2017.1/navigating-to-next-previous-error.html).